insertMoviesToDB.ts

```diff
@@ -0,0 +1,36 @@
+import{ Movie } from "./model/Movie" 
+import fs from 'fs';
+
+export async function insertMoviesData(): Promise<void> {
+    const movies = JSON.parse(fs.readFileSync('./movies.json', 'utf8'));
+  
+    for (const movie of movies) {
```

Comment:
if you use 
`const promises = movies.map(async movie=>{
    // rest of the code
    await newMovie.save();
})

await Promise.all(promises);`
The movies will be added in parallel and that will speed up the process.